tests/qunit/editor/test.js

```diff
@@ -8,8 +8,10 @@ export default class EditorTest extends EditorBase {
 		super( options );
 
 		QUnit.testStart( ( { module, name } ) => {
-			if ( this.$previewElementorEl ) {
-				this.$previewElementorEl.empty();
+			const $element = elementor.documents.getCurrent().$element;
```

Comment:
@matipojo it was changed since, before this PR, we didn't show the deprecation messages in the QUnit, but now we do, but, too many messages will cause QUnit to disconnected somehow, so I changed it here.